modules/nodes-descheduler-policies.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-policies_{context}"]
+= Descheduler policies
+
+The following descheduler policies are available:
+
+// TODO: the example YAML probably aren't right because the operator's config is slightly different than the orig Kubernetes config.
+
+////
+Questions:
+ - refer to these as policies or strategies? scheduler says policies, but the field used here is strategies.
+ - can you configure more than one at the same time?
+    - if you can, is order important?
+ - upstream kube says they're all enabled by default, but none are enabled by default on operator?
+ - does only lownodeutilization have additional config fields? how to find out?
+////
+
+Low node utilization::
+The `lownodeutilization` strategy finds nodes that are underutilized and evicts Pods, if possible, from other nodes in the hope that recreation of evicted Pods will be scheduled on these underutilized nodes.
++
+The underutilization of nodes is determined by several configurable parameters, such as CPU, memory, and number of Pods.
+If a node's usage is below the configured threshold for all parameters (CPU, memory, and number of Pods), then the node is considered to be underutilized.
++
+// TODO: need to describe the target threshold options. I don't understand when they're necessary. aren't they considered find anyways as long as they're not below, so what is setting a "max" doing?
```

Comment:
the threshold options allow you to define "busy" nodes and "empty" nodes

`lownodeutilization` looks for busy nodes and empty nodes, and tries to evict pods from the busy nodes (but not more than the empty nodes can theoretically handle without becoming "busy"). Note, it doesn't actually try to put the evicted pods on to the lower usage nodes (that's the scheduler's job), but it uses the `threshold` values to estimate what pods could fit. 

any node over the `targetThreshold` values is a potential eviction victim, and any node under the `threshold` values are assumed to be underutilized. but again there's no guarantee with this strategy that the pods will actually end up on those underutilized nodes

Reply:
That makes sense, thanks!